cmd/image/qcow2ova/qcow2ova.go

```diff
@@ -232,7 +232,7 @@ Qcow2 images location:
 			return err
 		}
 
-		rawImg := filepath.Join(ovaImgDir, ova.VolNameRaw)
+		rawImg := filepath.Join(ovaImgDir, fmt.Sprintf("%s-%s", opt.ImageName, ova.VolNameRaw))
```

Comment:
Instead of using the same pattern in so many places, you can create a variable here and use it everywhere(same variable can be passed to the `CreateTarArchive` function as well)